content/ADR-25-explorer-repositories-decoupling-.md

```diff
@@ -65,46 +61,45 @@ All the code bound to a web browser is contained exclusively inside this shell,
 
 The CI/CD of this repository is going to output internal and production releases for play.decentraland.zone and play.decentraland.org respectively.
 
-* Contains React and the entry point that instantiates Renderer and Kernel. 
+- Contains React and the entry point that instantiates Renderer and Kernel.
 
-* The typescript project will also be responsible for creating the ethereum provider and passing it as a parameter to Kernel.
+- The typescript project will also be responsible for creating the ethereum provider and passing it as a parameter to Kernel.
 
-* The typescript project contains a thin layer with the `unity-interface` and all the code that is directly related with the browser renderer (GIFs, mouse events wrapper, audio, video, etc).
+- The typescript project contains a thin layer with the `unity-interface` and all the code that is directly related with the browser renderer (GIFs, mouse events wrapper, audio, video, etc).
 
-* An independent Unity project contains the Renderer Package, the web specific entry point (InitialScene + Main + etc) and any web specific components that may be needed.
+- An independent Unity project contains the Renderer Package, the web specific entry point (InitialScene + Main + etc) and any web specific components that may be needed.
 
 ![resources/ADR-25-explorer-repositories-decoupling/image-2.svg](resources/ADR-25-explorer-repositories-decoupling/image-2.svg)
 
 ### Desktop Shell Repository
 
-Contains all the components to run a desktop version of Decentraland. 
+Contains all the components to run a desktop version of Decentraland.
 
 The CI/CD of this repository is going to be bound to internal and production releases that will be served from a link in the download section of our landing pages (TBD).
 
-* The Unity executable will be run by a native JS virtual machine (i.e. NodeJS) to run the scenes and some parts of the kernel. This will be done via a shell command, and this means two processes will be running by this stage. The JS project will communicate with the Unity process using IPC (Exact definition TBD) or FFI linking. In the future there will be different executables generated for each of the desktop platforms (win, mac, linux).
+- The Unity executable will be run by a native JS virtual machine (i.e. NodeJS) to run the scenes and some parts of the kernel. This will be done via a shell command, and this means two processes will be running by this stage. The JS project will communicate with the Unity process using IPC (Exact definition TBD) or FFI linking. In the future there will be different executables generated for each of the desktop platforms (win, mac, linux).
```

Comment:
```suggestion
- The Unity executable will be run by a native JS virtual machine (i.e. NodeJS) to run the scenes and some parts of the kernel. This will be done via a shell command, which means two processes will be running at this stage. The JS project will communicate with the Unity process using IPC (Exact definition TBD) or FFI linking. In the future, different executables will be generated for each desktop platform (win, mac, Linux).
```